public/app/features/alerting/unified/RuleEditorExisting.test.tsx

```diff
@@ -68,21 +67,6 @@ describe('RuleEditor grafana managed rules', () => {
       AccessControlAction.AlertingRuleExternalRead,
       AccessControlAction.AlertingRuleExternalWrite,
     ]);
-  });
-
-  it('can edit grafana managed rule', async () => {
```

Comment:
Diff looks weird but I just moved some pieces out into the beforeEach - the test is still there down below 👇